src/visitpy/visitpy/PyView3DAttributes.C

```diff
@@ -1170,173 +1170,6 @@ View3DAttributes_str(PyObject *v)
     return PyString_FromString(PyView3DAttributes_ToString(obj->data,"").c_str());
 }
 
-
-/*
```

Comment:
We may need to have @cyrush take a quick look here. The upper portion is actually a large comment embedded between C language comment designators `/*` and `*/`. The lower block is real code though. All I did as far as this specific `PyXxx.C` file is concerned though was run `make gen_python_all`

Reply:
Strange, View2D/3DAttributes should be skipped by the gen_python_all according to the notes in visitpy/CMakeLists.txt

Reply:
@biagas I am looking for the list of *all* `PyXXX.C` files I may have accidentally corrupted doing a `make gen_python_all`. I can't recall, did you suggest a starting point somewhere?

Reply:
> Strange, View2D/3DAttributes should be skipped by the gen_python_all according to the notes in visitpy/CMakeLists.txt

Could it be that even though the CMake comment designator is used for some of these objects, because it appears *inside* the list being passed to a `SET(...)` call, it is effectively being ignored and so lines that appear to be commented out there CMake still sees as valid entries in the list being set for `PY_GEN_STATE_OBJECTS` variable?

Reply:
@biagas Ok. I think I know what happened. I am pretty sure I caused this. I think `make gen_python_all` is working as you would expect. However, if you will recall, back there in 1076f4b1972698f35954dcfa8ca22419232e3699 I was unaware of `make gen_python_all` and had used a [custom shell script](https://github.com/visit-dav/visit/pull/5750#issuecomment-857089374) to find all `PyXXX.C` files that needed regeneration and then regenerated from that. So, that is where things went awry here and several files you were not expecting to get changed or to be changed so dramatically were changed. I will unravel this mess. I might wind up closing this PR and creating a new one.